src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -97,51 +98,51 @@ relations:
   - skos:closeMatch
   - skos:relatedMatch
 global_metadata:
-  subject_source: http://purl.obolibrary.org/obo/mondo.owl
+  subject_source: infores:mondo
   license: "https://creativecommons.org/publicdomain/zero/1.0/"
   mapping_date: "2017-09-20"
   mapping_set_id: "GLOBAL" # Apparently this is the one that matters.
 source_metadata:
   - from: MONDO
     to: OMIM
     metadata:
-      object_source: https://www.omim.org/
+      object_source: infores:omim
       object_source_version: http://www.orphadata.org/data/ORDO/ordo_orphanet_3.1.owl
       mapping_date: "2022-02-11"
       comment: "The latest OMIM is integrated with every Mondo release."
   - from: MONDO
     to: UMLS
     metadata:
-      object_source: https://uts.nlm.nih.gov/uts/
+      object_source: infores:umls
   - from: MONDO
     to: EFO
     metadata:
-      object_source: http://www.ebi.ac.uk/efo/efo.owl
+      object_source: infores:efo
   - from: MONDO
     to: ICD10CM
     metadata:
-      object_source: https://www.icd10data.com/
+      object_source: infores:icd10cm
       mapping_date: "2017-09-20"
   - from: MONDO
     to: Orphanet
     metadata:
-      object_source: https://www.orpha.net/
+      object_source: infores:orphanet
       object_source_version: http://www.orphadata.org/data/ORDO/ordo_orphanet_3.1.owl
       mapping_date: "2021-07-06"
       comment: "The latest Orphanet is integrated with every Mondo release."
   - from: MONDO
     to: NCIT
     metadata:
-      object_source: http://purl.obolibrary.org/obo/ncit.owl
+      object_source: infores:ncit
       object_source_version: http://purl.obolibrary.org/obo/ncit/releases/2018-06-14/ncit.owl
       mapping_date: "2018-06-14"
   - from: MONDO
     to: DOID
     metadata:
-      object_source: http://purl.obolibrary.org/obo/doid.owl
+      object_source: infores:disease-ontology
       object_source_version: http://purl.obolibrary.org/obo/doid/releases/2018-07-05/doid.owl
       mapping_date: "2018-07-05"
   - from: MONDO
     to: icd11.foundation
     metadata:
-      object_source: http://purl.obolibrary.org/obo/mondo/sources/icd11foundation.owl
+      object_source: infores:icd11-foundation
```

Comment:
should this be `infores:icd11-foundation` or `infores:icd11.foundation`?

Reply:
The resource does not technically exist yet 😜 https://github.com/biolink/information-resource-registry/pull/59

But the convention is in that resource is using a dash